waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -40,6 +40,19 @@
             (is (<= initial-rs-snapshot-version rs-snapshot-version'))
             (is (< rs-snapshot-version' rs-watch-version'))))))))
 
+(deftest ^:parallel ^:integration-slow test-kubernetes-custom-image
+  (testing-using-waiter-url
+    (when (using-k8s? waiter-url)
+      (let [{:keys [body]} (make-kitchen-request
+                                          waiter-url
+                                          {:x-waiter-name (rand-name)
+                                           :x-waiter-image "twosigma/integration"
+                                           :x-waiter-cmd "cd /tmp && python3 -m http.server $PORT0"
```

Comment:
You put the canary file in `/tmp`? You should probably add a comment explaining how/why this test works.

In hindsight, I would have preferred that we add a custom script to the `$PATH` in the test image, and then the command could have simply been `run-custom-image-test-server` or something like that. (The script would have handled the logic for starting the server with the custom content.) But it's probably not worth refactoring the docker image at this point.